.github/actions/flatpak-builder-lint/action.yaml

```diff
@@ -7,6 +7,10 @@ inputs:
   path:
     description: Path to flatpak-builder manifest or Flatpak build directory
     required: true
+  validateToPublish:
+    description: If false, turns some errors to non-errors for non-publish workflow
+    required: false
+    default: true
```

Comment:
What's the purpose of this input, if all consumers pass `false`?

Reply:
Wrong, in `publish.yaml` all consumers pass implicitly `true`

Reply:
If you feel like the two last commit are unnecessary, they just try to avoid encountering potential errors only while publishing.
Feel free to say it.

Reply:
It doesn't disable some test, it replace how the Github action should behave with some errors.

![Screenshot from 2023-11-08 14-00-00](https://github.com/obsproject/obs-studio/assets/17492366/71029dca-4271-4d46-959b-99c29b59bf05)

Reply:
> why is that undesireable?

I just realized that I misread you 🤦. You are in fact okay with what the two last commit try to implement.
Stupid me.